app/components/submission/ManuscriptMetadata/ManuscriptMetadata.js

```diff
@@ -63,45 +45,65 @@ const ManuscriptMetadata = ({ values, setFieldValue, setFieldTouched }) => (
       />
     </Box>
 
-    <CheckboxGeneratedChild
-      checkboxLabel="This manuscript has been discussed previously with an eLife editor"
-      fieldName="submissionMeta.discussedPreviously"
-      values={values}
+    <OptionalSection
+      label="This manuscript has been discussed with an eLife editor"
+      namedAs="discussion"
+      onClose={() => setFieldValue('submissionMeta.discussion', null)}
+      onOpen={() => setFieldValue('submissionMeta.discussion', '')}
+      open={values.submissionMeta.discussion !== null}
     >
       <ValidatedField
         component={Textarea}
-        label="What was discussed and with whom"
+        label="What did you discuss and with whom?"
         name="submissionMeta.discussion"
       />
-    </CheckboxGeneratedChild>
+    </OptionalSection>
 
-    <CheckboxGeneratedChild
-      checkboxLabel="This manuscript has been considered by eLife previously"
-      fieldName="submissionMeta.consideredPreviously"
-      values={values}
+    <OptionalSection
+      label="This manuscript has been previously considered by eLife"
+      namedAs="previousArticle"
+      onClose={() => setFieldValue('submissionMeta.previousArticle', null)}
+      onOpen={() => setFieldValue('submissionMeta.previousArticle', '')}
+      open={values.submissionMeta.previousArticle !== null}
     >
       <ValidatedField
-        label="Article title/reference No"
+        label="Article title"
         name="submissionMeta.previousArticle"
       />
-    </CheckboxGeneratedChild>
+    </OptionalSection>
 
-    <CheckboxGeneratedChild
-      checkboxLabel="This manuscript is a co-submission"
-      fieldName="submissionMeta.cosubmission"
-      values={values}
+    <OptionalSection
+      label="This manuscript is a co-submission"
+      onClose={() => setFieldValue('submissionMeta.cosubmission', [])}
+      onOpen={() =>
+        setFieldValue('submissionMeta.cosubmission', [{ title: '' }])
+      }
+      open={values.submissionMeta.cosubmission.length}
     >
-      <div>
-        <ValidatedField
-          label="Article title"
-          name="submissionMeta.cosubmissionTitle"
-        />
-        <ValidatedField
-          label="Reference No"
-          name="submissionMeta.cosubmissionId"
-        />
-      </div>
-    </CheckboxGeneratedChild>
+      {values.submissionMeta.cosubmission.map((_, rowIndex) => (
+        // eslint-disable-next-line react/no-array-index-key
+        <Box key={rowIndex} mb={2}>
+          <ValidatedField
+            label="Article title"
```

Comment:
I think we said that the label for the second field should have "(optional)" appended.

Reply:
Yup @hdrury1 confirmed